lychee-lib/src/client.rs

```diff
@@ -447,7 +447,7 @@ mod test {
             .build()
             .client()
             .unwrap();
-        let res = client.check("https://example.org").await.unwrap();
+        let res = client.check("http://example.org").await.unwrap();
```

Comment:
🤦‍♀️ I used the global search and replace. Should have checked what the test was actually doing. 😄